domain/src/main/kotlin/no/nav/su/se/bakover/domain/vedtak/Vedtak.kt

```diff
@@ -31,7 +43,7 @@ sealed class Vedtak {
         val saksbehandler: NavIdentBruker.Saksbehandler,
         val attestant: NavIdentBruker.Attestant,
         val utbetalingId: UUID30,
-        val eksterneIverksettingsteg: EksterneIverksettingsstegEtterUtbetaling,
+        override val eksterneIverksettingsteg: JournalføringOgBrevdistribusjon,
```

Comment:
Siden denne nå ligger på vedtaket, kunne vi vurdert å fjerne den herfra? Da hadde vi sluppet å oppdatere den 2 plasser. Og det er jo noe som kanskje hører mer til vedtaket enn behandlinga? Kanskje behandlinga er ferdig og "låst" når den har kommet i IVERKSATT tilstanden? En kunne jo sett på å gjøre det samme med utbetalingId?

Reply:
Jepp, se kommentaren min angående forslag til videre arbeid. Unngikk å ta det med her for å begrense scopet noe.